public/script.js

```diff
@@ -4214,12 +4214,15 @@ export async function Generate(type, { automatic_trigger, force_name2, quiet_pro
             ? substituteParams(jailbreak, name1, name2, (power_user.sysprompt.post_history ?? ''))
             : baseChatReplace(power_user.sysprompt.post_history, name1, name2);
 
-        // When continuing generation of previous output, last user message precedes the message to continue
-        if (isContinue) {
-            coreChat.splice(coreChat.length - 1, 0, { mes: jailbreak, is_user: true });
-        }
-        else {
-            coreChat.push({ mes: jailbreak, is_user: true });
+        // Only inject the jb if there is one
+        if (jailbreak && typeof(jailbreak) != 'undefined') {
```

Comment:
Comparing with undefined should be done via `!== 'undefined'`.
But isn't `if (jailbreak)` enough here? That checks for a non-empty string, and also excludes null and undefined.